FreeRTOS-Plus/Demo/FreeRTOS_Plus_TCP_Minimal_Windows_Simulator/DemoTasks/TCPEchoClient_SingleTasks.c

```diff
@@ -250,12 +256,17 @@ TickType_t xTimeOnEntering;
 					{
 						/* The echo reply was received without error. */
 						ulTxRxCycles[ xInstance ]++;
+
+						/* The "Received correct data" line is used to determine if
+							* this demo runs as part of a GitHub workflow. */

```

Comment:
This comment is misleading - the line is used to determine if the demo runs successfully on a GitHub workflow